platform/kube/controller.go

```diff
@@ -298,6 +399,22 @@ func (c *Controller) List(kind, namespace string) (map[model.Key]proto.Message,
 	return out, errs
 }
 
+func (c *Controller) listIngresses(kind, namespace string) (map[model.Key]proto.Message, error) {
```

Comment:
Well, you're right that the "kind" parameter isn't really needed here. I left it for uniformity with the other listX methods.